src/data/finset.lean

```diff
@@ -65,6 +65,9 @@ ext.2
 @[simp] theorem coe_inj {s₁ s₂ : finset α} : (↑s₁ : set α) = ↑s₂ ↔ s₁ = s₂ :=
 (set.ext_iff _ _).trans ext.symm
 
+lemma to_set_injective {α} : function.injective (finset.to_set : finset α → set α) :=
```

Comment:
Can this be changed to `coe_injective` with statement `function.injective (coe : finset A -> set A)`?